src/Base/index.js

```diff
@@ -1,78 +1,93 @@
 import React from 'react';
-import {ActivityIndicator, Text, View, Pressable} from 'react-native';
+import {Text, View, TextInput} from 'react-native';
 
-const Button = ({
-  disabled = false,
-  onPress,
-  busy = false,
-  loaderIcon = <ActivityIndicator color="#fff" />,
-  busyText = 'Processing..',
+const Input = ({
+  onChangeText,
+  iconPosition,
+  icon,
+  style,
+  value,
+  placeholder = 'Enter Text',
+  label = null,
+  error,
   ...props
 }) => {
+  const [focused, setFocused] = React.useState(false);
   const textStyles = {
     ...(props.textStyles || {
```

Comment:
This should be renamed to props.textInputStyles

The textStyles name is used in buttons package, is literally for <Text>, here we have TextInput, so naming of this is going to be important